### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'raise_for_status'

### ISSUE THREAD

**Issue by ddebrunner** [IBMStreams/streamsx.topology#885]
Running:

python3 -m unittest test2_spl.TestBluemixSPL

but think it occurs with any bluemix test.
```
ERROR: test_SPLBeaconFilter (test2_spl.TestBluemixSPL)
Test a Source and a Map operator.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/streamsadmin/topology/42test/streamsx.topology/test/python/topology/test2_spl.py", line 38, in test_SPLBeaconFilter
    tester.test(self.test_ctxtype, self.test_config)
  File "/home/streamsadmin/topology/42test/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py", line 250, in test
    passed = self._streaming_analytics_test(ctxtype, config)
  File "/home/streamsadmin/topology/42test/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py", line 289, in _streaming_analytics_test
    self.sc = StreamingAnalyticsConnection(service_name=config[ConfigParams.SERVICE_NAME])
  File "/home/streamsadmin/topology/42test/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py", line 158, in __init__
    rest_api_url = _get_rest_api_url_from_creds(self.credentials)
  File "/home/streamsadmin/topology/42test/streamsx.topology/com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py", line 248, in _get_rest_api_url_from_creds
```

**Comment by ddebrunner** [comment]
@wmarshall484 @ibmkendrick 

May be due to an interaction between your latest PRs.

#870 and #878

**Comment by wmarshall484** [comment]
Fixed in #887
